recipes/application_python.rb

```diff
@@ -95,6 +95,7 @@
 if node.deep_fetch(stackname, 'code-deployment', 'enabled')
   node[stackname][node[stackname]['webserver']]['sites'].each do |port, sites|
     sites.each do |site_name, site_opts|
+      next if site_opts['repository'] == ''
```

Comment:
@prometheanfire I'm confused... if `site_opts.key?('repository')` is false (if it's unset), `site_opts['repository']` still evaluates to `false` when coerced into a boolean value. It won't throw a Nil exception. Only accessing a subkey would throw an error, like `site_opts['repository']['foo']`.